### ISSUE TITLE
ConResAnalyzer: Samples

### ISSUE THREAD

**Issue by SMotaal** [ConRes/experimental#2]
Scanned samples will be needed to help with POC work (see #1).

- [ ] Scanned Samples

  For now each scanned target needs the TDF file and the **cropped** block images.

  - Case folder (`‹…›/`)
    - TDF file (`‹…›/‹…›.tdf`)
    - Block files (`‹…›/‹…› - Block X.png`)

**Comment by FranzSigg** [comment]
Are the scans not also in a subfolder of the case folder?
Do you need the EPS target file in the Case folder, or is it enough to have the PDF of the target in the Case folder?

**Comment by SMotaal** [comment]
Certainly, complete case folders will include all the trimmings, but maybe for now let's just keep things agile and only keep the `.tdf` and block files… sounds good?

**Comment by SMotaal** [comment]
While we start off with just blocks, TDF and a `case.json` (which the app introduces)… thinking this through, it might be a good idea to consider "Scanned Blocks" and/or "Cropped Blocks" folders… etc. so that a loader expects a more concrete set of folders and files.

All in due time though.
